checkstyle/suppressions.xml

```diff
@@ -158,7 +158,7 @@
               files="StreamThread.java"/>
 
     <suppress checks="ClassDataAbstractionCoupling"
-              files="(KStreamImpl|KTableImpl).java"/>
+              files="(KStreamImpl|KTableImpl|WorkerSourceTaskTest).java"/>
```

Comment:
Instead of adding this to Kafka Streams rules, can we move it to the Connect section just above?